README.md

```diff
@@ -1,7 +1,7 @@
 # Artifacts Poetry Plugin
 
 ## Description
-The Artifacts Poetry Plugin is used to print out all direct and transitive package dependencies of a python project.
+The Artifacts Poetry Plugin is used to deploy all direct and transitive package dependencies of a python project to an alternate repository. The plugin requires the use of a pyproject.toml and generated poetry.lock file. All dependencies must be installed poetry's local artifacts cache. 
```

Comment:
possible grammar update: "...must be installed _in_ poetry's local..."

Reply:
Fixed.